website/docs/fun/qotd/advanced/overview.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Overview
+---
+
+An advanced QOTD system which automatically posts a question every day, either a random default question or from a queue
+of suggested questions from your server members.
+
+## Features
+
+- Permits server members to add questions to a queue.
+  - Allows you to configure a queue channel to view the question queue and submit new questions.
```

Comment:
As a general comment, this overview seems to be directed toward both administrators (configuration) and users (how to submit questions.) I have no strong opinion on whether the user guidance is appropriate and would be happy to keep it in, but wonder whether a clearer separation would be beneficial.

Reply:
How's this?

Reply:
Nice, LGTM.